geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -70,28 +77,74 @@
 @RunWith(GeodeParamsRunner.class)
 public class P2PMessagingConcurrencyDUnitTest {
 
-  // how many messages will each sender generate?
-  private static final int MESSAGES_PER_SENDER = 1_000;
+  // how many sending member JVMs
+  private static final int SENDERS = 1;
 
-  // number of concurrent (sending) tasks to run
-  private static final int SENDER_COUNT = 10;
+  // number of concurrent (sending) tasks to run in each sending JVM
+  private static final int TASKS_PER_SENDER = 10;
 
-  // (exclusive) upper bound of random message size, in bytes
-  private static final int LARGEST_MESSAGE_BOUND = 32 * 1024 + 2; // 32KiB + 2
+  // how many messages will each sending task generate?
+  private static final int MESSAGES_PER_SENDING_TASK = 1_000;
+
+  /*
+   * Upper bound (exclusive) of random message size, in bytes.
+   * The magnitude is chosen to give us a mix of messages around
+   * 16KB. I want a number that results in a largest message that
+   * is not a power of two since I believe, for no particular reason,
+   * that it might catch more bugs than a power of two would,
+   * so I add 2.
+   */
+  private static final int LARGEST_MESSAGE_BOUND = 32 * 1024 + 2;
+
+  /*
+   * Non-random payload content makes debugging easier when a NULL cipher is used.
+   * Null ciphers are not allowed on TLSv1.3 though.
+   */
+  private static boolean RANDOMIZE_PAYLOAD_CONTENT = true;
```

Comment:
What is the point of this flag? Should this not be made into a separate test or test parameter?

Reply:
Random content arguably makes for a stronger test. Generally random content catches bugs that uniform/repeated content does not. That's why the test initially randomized the content.

But in troubleshooting a TLS test, on protocol versions before TLSv1.3 it is sometimes helpful to use a NULL cipher so you can distinguish payload content from non-payload content at a glance e.g. in packet captures. That's why this feature (to turn off randomization) was introduced. It was useful enough that I thought it was valuable to keep it behind this feature flag.

Reply:
reverted changes to this test